software/daily-stars-explorer.yml

```
@@ -0,0 +1,14 @@
+name: "Daily Stars Explorer"
+website_url: "https://emanuelef.github.io/daily-stars-explorer"
+source_code_url: "https://github.com/emanuelef/daily-stars-explorer"
+description: "Track GitHub repo trends with daily star insights to see growth and community interest over time."
+licenses:
+  - MIT
+platforms:
+  - Go
+  - Javascript
+  - Docker
```

I cant seem to find JS nor Go install instructions.
```suggestion
platforms:
  - Docker
```

---

Ok, I can improve the JS documentation but to self host just the docker image would suffice.

---

Yep, Docker is great! For a platform to be added, there needs to be install instructions.

---

Added some instructions to run the backend in Go and frontend in JS.